python/vtool/houdini_lib/core.py

```diff
@@ -7,3 +10,43 @@ def save(filepath):
 
 def load(filepath):
     hou.hipFile.load(filepath)
+
+
+def merge(filepath):
+    hou.hipFile.merge(filepath)
+
+def export_nodes(path, nodes):
+
+    util_file.refresh_dir(path, delete_directory=False)
+
+    parent = nodes[0].parent()
+    parent_path = nodes[0].path()
+    parent_path = util_file.get_dirname(parent_path)
+    if parent_path.startswith('/'):
```

Comment:
This is a pointless conditional check for something you are mutating anyways. Just use a left strip and remove the path separator from the start and remove the if statement.

Reply:
I'm using the houdini node path which starts with '/' . If I do path1 = 'C:/something' and path2 = '/obj/houdini_something' and do a os.path.join(path1, path2) it returns 'C:/obj/houdini_something' instead of 'C:/something/obj/houdini_something'.

Reply:
os.path.join('C:/something', '/obj/houdini_something'.lstrip("/"))